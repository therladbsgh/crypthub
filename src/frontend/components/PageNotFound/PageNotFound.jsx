import React, { Component } from "react";

export default class PageNotFound extends Component {
	render() {
		return (
	  		<div id="pageNotFound">
				Error 404: Page not found
	  		</div>
		);
  	}
}
