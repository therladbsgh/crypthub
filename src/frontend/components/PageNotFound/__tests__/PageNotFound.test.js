import React from 'react';
import PageNotFound from '../PageNotFound';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  	const tree = renderer.create(<PageNotFound />).toJSON();
  	expect(tree).toMatchSnapshot();
});