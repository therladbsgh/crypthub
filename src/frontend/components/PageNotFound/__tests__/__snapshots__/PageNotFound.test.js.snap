// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  id="pageNotFound"
>
  Error 404: Page not found
</div>
`;
